<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InteractiveEdgeDetectionDialog</class>
 <widget class="QMainWindow" name="InteractiveEdgeDetectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="pixmapLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Masque</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QFormLayout" name="formLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="maskTypeLabel">
             <property name="text">
              <string>Masque à appliquer :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="maskTypeComboBox">
             <item>
              <property name="text">
               <string>Prewitt</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Sobel</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Kirsch</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="filteringTypeLabel">
             <property name="text">
              <string>Filtrage :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="filteringTypeComboBox">
             <item>
              <property name="text">
               <string>Bi-directionnel</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Multi-directionnel</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Seuillage par hystérisis</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Seuil haut :</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSlider" name="highThresholdSlider">
           <property name="value">
            <number>20</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
           <property name="tickInterval">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="highThresholdLineEdit">
           <property name="suffix">
            <string>%</string>
           </property>
           <property name="value">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Seuil bas :</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="lowThresholdSlider">
           <property name="value">
            <number>10</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
           <property name="tickInterval">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="lowThresholdLineEdit">
           <property name="suffix">
            <string>%</string>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>652</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionEnregistrer_sous"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEnregistrer_sous">
   <property name="text">
    <string>Enregistrer sous...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>highThresholdSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>highThresholdLineEdit</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>highThresholdLineEdit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>highThresholdSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>602</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lowThresholdSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lowThresholdLineEdit</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lowThresholdLineEdit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lowThresholdSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>602</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
